src/main/java/mod/gate/mixin/InGameHudMixin.java

```diff
@@ -0,0 +1,35 @@
+package mod.gate.mixin;
+
+
+import mod.gate.events.ActionbarReceiveEvent;
+import mod.gate.events.EventHandler;
+import net.minecraft.client.gui.hud.InGameHud;
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(InGameHud.class)
+public class InGameHudMixin {
+
+    private static String cachedContent;
+
+
+    @Inject(at = @At("HEAD"), method = "setOverlayMessage")
+    private void setOverlayMessageMixin(Text message, boolean tinted, CallbackInfo ci) {
+        if (message.asString() != cachedContent) {
```

Comment:
use `!message.asString().equals(cachedContent)` since my IDE will complain about it otherwise since it has some sort of anti-null check